WeatherPlannerDemo/UI/ViewControllers/CityDetailsViewController/CityDetailsViewController.swift

```diff
@@ -0,0 +1,80 @@
+import UIKit
+
+class CityDetailsViewController: UIViewController {
+    
+    var cityDetailsTableView: UITableView!
+    
+    let cityDetailsViewModelList: CityDetailsViewModelList
+    
+    init(cityViewModel: CityViewModel) {
+        cityDetailsViewModelList = CityDetailsViewModelList(cityViewModel: cityViewModel)
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        buildViews()
+        
+        cityDetailsTableView.delegate = self
+        cityDetailsTableView.dataSource = self
+        
+        cityDetailsViewModelList.delegate = self
+        
+        registerTableViewCells()
+        
+        cityDetailsViewModelList.fetchPollutionInfo()
+    }
+    
+    private func registerTableViewCells() {
+        cityDetailsTableView.register(
+            CityDetailsTableViewCell.self,
+            forCellReuseIdentifier: CityDetailsTableViewCell.reuseIdentifier
+        )
```

Comment:
For function call use this format:
```
cityDetailsTableView.register(
             CityDetailsTableViewCell.self,
             forCellReuseIdentifier: CityDetailsTableViewCell.reuseIdentifier)
```
Check this [here](https://github.com/fiveagency/ios-swift-guide/tree/master/Code%20Style#function-declarations) in our style guide.